test/integration/sapi-read-clock.int.c

```diff
@@ -16,6 +16,7 @@
 
 #define LOGMODULE test
 #include "util/log.h"
+#include "test-esapi.h"
```

Comment:
Is this for EXIT_SKIP only?
I would prefer to use 77 magic number instead of including esapi header here. I will fix that. Thanks